src/routeApp/validations.js

```diff
@@ -0,0 +1,65 @@
+import { body } from 'express-validator';
+
+const routeValidation = () => [
+	body('origin', 'origin_required').notEmpty(),
+	body('destination', 'destination_required').notEmpty(),
+	body('distance', 'distance_required')
+		.notEmpty()
+		.isNumeric()
+		.withMessage('number')
+		.isLength({
+			max: 5,
+		})
+		.withMessage('greater_5'),
+	body('start_coor', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+	body('end_coor', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+	body('stop_points', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+];
+export const editValidation = () => [
+	body('origin', 'origin_required').optional().notEmpty(),
+	body('destination').optional().notEmpty().withMessage('destination_required'),
+	body('distance')
+		.notEmpty()
+		.withMessage('distance_required')
+		.isNumeric()
+		.withMessage('number')
+		.isLength({
+			max: 5,
+		})
+		.withMessage('greater_5'),
+	body('start_coor', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+	body('end_coor', 'geographical_coordinates')
+		.optional()
+		.isLength({
+			max: 20,
+		})
+		.withMessage('greater_10'),
+	body('stop_points', 'geographical_coordinates')
```

Comment:
Since this entry is for **Path** datatype, this validation should be removed instead check if the minimum length is above `30` characters. 

Path; A collection of geographical points 
Ex: **"(-1.9317026812342064, 30.063155494580016),(-1.92800534178346, 30.07661466134904)"**